FinniversKit/Sources/DNA/Images/ImageProvider.swift

```diff
@@ -121,4 +121,8 @@ public struct DefaultImageProvider: ImageProvider {
     public var brandMapDirections: UIImage {
         UIImage(named: .mapDirections)
     }
+    
+    public var toriLogo: UIImage {
```

Comment:
I don't think adding this here will be correct. This is the default ImageProvider which serves Finn branded images. The Tori logo should be served by `brandLogo` on the ImageProvider implementation for Tori (conveniently named `ToriImageProvider` in the app repo).

Reply:
Thank you for your input! Has been resolved now 🚀